wandb/util.py

```diff
@@ -1477,3 +1477,18 @@ def check_dict_contains_nested_artifact(d, nested=False):
         ) and nested:
             return True
     return False
+
+
+def deprecate(field_name: str, warning_message: str):
```

Comment:
deprecate needs to take a run object (maybe optional)

Reply:
done